doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support intra-pod communications such as between all containers within a pod on localhost |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.09` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) CNF/VNF workloads. |
+| `req.inf.ntw.10` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.11` | Network | The SDN solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.12` | Network | The SDN solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.13` | Network | The Architecture **must** support dual stack IPv4 and IPv6 in Pods and Services. |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+
+<p align="center"><b>Table 2-2:</b> Containerised Platform: Infrastructure Requirements.</p>
+
+<a name="2.3.3"></a>
+### 2.3.3 Containerised Infrastructure Management (CIM)
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.cim.01` | General | The Architecture **must** allow infrastructure resource sharing. |
+| `req.cim.02` | General | The Architecture **must** allow discovery and management of NFVI resources. |
+| `req.cim.03` | General | The Architecture **must** support Enhanced Platform Awareness (EPA). |
```

Comment:
EPA is an Intel term and covers a not well defined set of technologies. I would be more happy if we would list the technologies as requirements instear of generalizing them to EPA.

- Support of hugepages
- NUMA resource CPU allocation
- SR-IOV support
- User space networking support

Reply:
@CsatariGergely As you know, EPA is now in OpenStack (contributed by Intel) also used by ONAP and concerns the hardware accelerations. The requirements can be modified to whatever is the general consensus

Reply:
> 
> 
> @CsatariGergely As you know, EPA is now in OpenStack (contributed by Intel) also used by ONAP and concerns the hardware accelerations. The requirements can be modified to whatever is the general consensus

Hmm interesting, EPA means different things even in the different Intel whitepapers. For [OpenStack](https://01.org/sites/default/files/page/openstack-epa_wp_fin.pdf) it is different accelerator technologies while for [Kubernetes](https://builders.intel.com/docs/networkbuilders/enhanced-platform-awareness-in-kubernetes-application-note.pdf) it is node discovery, CMK, huge pages. One more reason to list the specific technologies.

Reply:
@CsatariGergely good catch, interesting but by the end of the day both definition cover performance and accelation techniques

Reply:
> 
> 
> @CsatariGergely good catch, interesting but by the end of the day both definition cover performance and accelation techniques

Yes, but here we should be precise about what do we include in the Reference Architecture.

Reply:
> 
> 
> > @CsatariGergely good catch, interesting but by the end of the day both definition cover performance and accelation techniques
> 
> Yes, but here we should be precise about what do we include in the Reference Architecture.

Yes, similar to text in RA1.

To ensure low integration effort key is also How it needs to be implemented (encapsulate intelligence inside SW platform vs exposing many APIs).

Reply:
> > @CsatariGergely good catch, interesting but by the end of the day both definition cover performance and accelation techniques
> 
> Yes, but here we should be precise about what do we include in the Reference Architecture.

@CsatariGergely , What did you think? can we use the well-defined definition for EPA ?

Reply:
@CsatariGergely @ASawwaf will be defined in RA-2 Ch 1 -- @tomkivlin

Reply:
@ASawwaf I do not know about a well-defined definition of EPA.

Reply:
@CsatariGergely , let me put a definition a common definition and let's agree on it 
@pgoyal01 shall we open an issue for it ?

Reply:
@ASawwaf please do, enter the Issue # here and resolve conversation. Thanks

Reply:
[RA-2 Ch02] Requirements- EPA definition #592